spec-main/api-web-contents-spec.ts

```diff
@@ -1245,6 +1245,35 @@ describe('webContents module', () => {
     });
   });
 
+  describe('crash behavior', () => {
+    let w: BrowserWindow;
+    beforeEach(async () => {
+      w = new BrowserWindow({ show: false, webPreferences: { nodeIntegration: true } });
+      await w.loadURL('data:text/html,<h1>Hello, World!</h1>');
```

Comment:
```suggestion
      await w.loadURL('about:blank');
```

should be fine yeah?